src/routes/Home.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Searchbar from '../components/Searchbar';
+import Logo from '../components/Logo';
+import Picture from '../components/Picture';
+
+const Home = () => {
+  const { filt } = useSelector((state) => state.apod);
+  return (
+    <div className="h-screen font-lato bg-blue-500">
+      <Searchbar />
+      <Logo />
```

Comment:
- [ ] Great on making the filteration but since here there is a Navbar so it would be better if you just make your navbar design just as the given wireframe so please fix it you can import icons from react icons or UI material both gives you great icons ✅💯🔥

### You can add it right away before the displayed images 
![image](https://user-images.githubusercontent.com/58553711/217970078-8c03efbb-efee-4cc3-90cd-d75ab283a047.png)


- Your design
![image](https://user-images.githubusercontent.com/58553711/217969946-316e4642-672e-4e1a-90d5-2c47f7a7eec6.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/217969838-98d9935e-3284-43cd-8c49-e09eff70fe3f.png)